api/Controllers/DashboardController.cs

```diff
@@ -74,5 +77,31 @@ public async Task<IActionResult> GetCourtCalendar(string startDate, string endDa
 
             return Ok(result);
         }
+
+        /// <summary>
+        /// Temp solution to allow viewing of other judge's calendar/court list.
+        /// </summary>
+        /// <returns>List of active judges.</returns>
+        [HttpGet]
+        [Route("judges")]
+        public async Task<IActionResult> GetJudges()
+        {
+            if (!this.User.CanViewOthersSchedule())
+            {
+                return Unauthorized();
+            }
+
+            try
+            {
+                var result = await _dashboardService.GetJudges();
+                return Ok(result);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine("Error retrieving judges.");
+                Console.WriteLine(ex);
+                return Ok(new List<PCSSModels.PersonSearchItem>());
```

Comment:
Should this return 404 Not found, or 500 Internal Server Error, along with an error message?